src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -2,9 +2,6 @@
 # Copyright Contributors to the OpenTimelineIO project
 
 from .. _otio import ( # noqa
-    # errors
-    CannotComputeAvailableRangeError,
```

Comment:
This is a potentially breaking change.

Reply:
you mean breaking for third parties, if they use it? Maybe open an issue as an RFC linked to this PR and if no one speaks up we can (1) document that it's removed in the change notes, (2) decide that the removal is fine.

Reply:
I added it back but it won't show up in the API reference. I will open a separate issue to remove it.